wire/internal/wire/wire_test.go

```diff
@@ -576,10 +584,28 @@ func (test *testCase) materialize(gopath string) error {
 	return nil
 }
 
-func runGo(bctx *build.Context, args ...string) error {
+// writeGoMod generates go.mod files for the test package and its dependency.
```

Comment:
Should these be checked in and verified? Or are they always going to be the same if wire_gen.go is the same as before?

Reply:
These should stay the same:
1) All test packages should be under "module example.com"
2) We are not going to have random third-party dependencies, and will likely only depend on the wire.go file.